packages/reporters/tracer/src/TracerReporter.js

```diff
@@ -66,14 +66,14 @@ export default (new Reporter({
       case 'buildSuccess':
       case 'buildFailure':
         nullthrows(tracer).flush();
+        tracer = null;
```

Comment:
@marcins This actually fixes a race condition that showed up in the tests where it's possible that the tracer isn't cleared quick enough.